CodeReview/src/commands/CFocus.h

```diff
@@ -38,10 +39,16 @@ class CODEREVIEW_API CFocus : public Interaction::Command
 
 		struct FocusInformation
 		{
-			enum FocusType {None, Center, Highlight};
+			enum FocusType {
+				None = 0x0,
+				Center = 0x1,
+				Highlight= 0x2
+			};
+
+			using FocusTypes = QFlags<FocusType>;
 
 			int step_{};
-			FocusType type_{};
+			FocusTypes type_{};
```

Comment:
The step above, should probably be `-1` by default, also indicating an invalid info.